workflows4s-web-ui/src/main/scala/workflows4s/web/ui/http/Http.scala

```diff
@@ -2,23 +2,89 @@ package workflows4s.web.ui.http
 
 import cats.effect.IO
 import sttp.client4.impl.cats.FetchCatsBackend
-import workflows4s.web.api.endpoints.WorkflowEndpoints
-import sttp.tapir.client.sttp4.SttpClientInterpreter
-import workflows4s.web.api.model.{WorkflowDefinition, WorkflowInstance}
+import sttp.client4.*
+import io.circe.parser.decode
+import workflows4s.web.api.model.{ProgressResponse, WorkflowDefinition, WorkflowInstance}
 
 object Http {
 
-  private val backend     = FetchCatsBackend[IO]()
-  private val interpreter = SttpClientInterpreter()
+  private val backend = FetchCatsBackend[IO]()
+  private val baseUrl = "http://localhost:8081"
 
   def getInstance(workflowId: String, instanceId: String): IO[WorkflowInstance] = {
-    val request = interpreter.toRequestThrowErrors(WorkflowEndpoints.getInstance, None)(workflowId, instanceId)
-    backend.send(request).map(_.body)
+    val uri = uri"$baseUrl/api/v1/definitions/$workflowId/instances/$instanceId"
+    basicRequest
+      .get(uri)
+      .send(backend)
+      .flatMap { response =>
+        response.body match {
+          case Right(json) =>
+            IO.fromEither(decode[WorkflowInstance](json))
+              .adaptError(err => new Exception(s"Failed to parse WorkflowInstance: ${err.getMessage}"))
+          case Left(error) =>
+            IO.raiseError(new Exception(s"HTTP Error: $error"))
+        }
+      }
   }
 
   def listDefinitions: IO[List[WorkflowDefinition]] = {
-    val request = interpreter.toRequestThrowErrors(WorkflowEndpoints.listDefinitions, None)(())
-    backend.send(request).map(_.body)
+    val uri = uri"$baseUrl/api/v1/definitions"
```

Comment:
what happened here? 

The goal of using tapir is having shared endpoint definitions that we can use on both server and client. 

This means we should never construct uris by hand but use those definitions as was there before. 

So lets revert here and adjust the new calls as well.